middleware/http/client_test.go

```diff
@@ -0,0 +1,52 @@
+package http_test
+
+import (
+	"net/http"
+	"testing"
+
+	zipkin "github.com/openzipkin/zipkin-go"
+	httpmiddleware "github.com/openzipkin/zipkin-go/middleware/http"
+	"github.com/openzipkin/zipkin-go/reporter/recorder"
+)
+
+func TestMiddleware(t *testing.T) {
+	reporter := recorder.NewReporter()
+	defer reporter.Close()
+
+	ep, _ := zipkin.NewEndpoint("httpClient", "")
```

Comment:
suppose "" is better than nil?

Reply:
Yes. nil would only work if changing string parameter to pointer to string which is not a good idea.